README.md

```diff
@@ -0,0 +1,153 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+
+  <h3><b>Bookstore</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Bookstore] <a name="about-project"></a>
+
+**[Bookstore]** is a React book app.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack">ReactJS</a>
+
+<details>
+  <summary>Client</summary>
+  React
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[To be implemented in the near future]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+- Create a local directory that you want to clone the repository.
+
+- Open the command prompt in the created directory.
+
+- On the terminal run this command git clone https://github.com/joshualyngdoh/bookstore.git
+
+- Go to the repository folder using command prompt cd bookstore.
+
+- Install the dev dependencies for linters run npm install.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Github account
+Vs Code
+
+### Setup
+
+Clone this repository to your desired folder:
+
+https://github.com/joshualyngdoh/bookstore.git
+
+
+### Install
+
+Install this project with: Github
+
+### Usage
+
+To run the project, execute the following command:
+
+https://github.com/joshualyngdoh/bookstore.git
+
+### Run tests
+
+To run tests, run the following command:
+
+- npx eslint "**/*.{js,jsx}"
+- npx stylelint "**/*.{css,scss}"
+
```

Comment:
- Please, kindly remove this section as you won't be writing test for the project to make your readme more professional.

Reply:
Thank you for the feedback.